Code/Framework/AzFramework/Platform/Android/AzFramework/Device/DeviceAttributesCommon_Android.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Android/JNI/JNI.h>
+#include <AzCore/Android/JNI/Object.h>
+#include <AzFramework/Device/DeviceAttributeDeviceModel.h>
+#include <AzFramework/Device/DeviceAttributeRAM.h>
+
+namespace AzFramework
+{
+    DeviceAttributeDeviceModel::DeviceAttributeDeviceModel()
+    {
+        static constexpr const char* JavaFieldName = "MODEL";
+        AZ::Android::JNI::Object obj("android/os/Build");
+        obj.RegisterStaticField(JavaFieldName, "Ljava/lang/String;");
+        m_value = obj.GetStaticStringField(JavaFieldName);

```

Comment:
Maybe we should print this info out for debugging purposes. Thoughts?

Reply:
I was doing that for testing, but removed it from this location - maybe a better place would be in the system component, that way it could print consistently on all platforms - or we could have a single function that enumerated and printed out all device attributes that could be controlled with a cvar.